packages/itwinui-react/src/core/Radio/Radio.tsx

```diff
@@ -17,6 +17,10 @@ type RadioProps = {
    * Status of the radio.
    */
   status?: 'positive' | 'warning' | 'negative';
+  /**
+   * Passes props to Radio label.
+   */
+  labelProps?: React.ComponentProps<'span'>;
```

Comment:
Radio also needs wrapper props (similar to Checkbox)

Reply:
Added.